File: 백준/Silver/1764. 듣보잡/듣보잡.js
const input = require("fs").readFileSync(0).toString().split("\n");
const [N, M] = input[0].split(" ").map(a => +a);
const noListen = new Set(input.slice(1, N + 1));
const noSee = input.slice(N + 1);
const noName = [];
let cnt = 0;
noSee.forEach(el => {
    if (noListen.has(el)) {
        noName.push(el);
        cnt++;
    }
});
noName.sort();
console.log(cnt);
console.log(noName.join("\n"));

File: 백준/Silver/1764. 듣보잡/듣보잡.py
import sys
input = sys.stdin.readline
n,m = map(int, input().split())
no_hear = {input() for i in range(n)}
no_see = {input() for i in range(m)}

intersection = sorted(no_hear & no_see)

print(len(intersection))
print("".join(intersection))